query/language/match.feature

```diff
@@ -2152,7 +2152,7 @@ Feature: TypeQL Match Clause
       | attr:second-name:"Morgan"        |
       | attr:passport-first-name:"Alice" |
 
-    Then typeql read query; fails with a message containing: "empty-set for some variable"
+    Then typeql read query; fails with a message containing: "Type-inference was unable to find compatible types for the pair of variables"
```

Comment:
maybe one example in match and one in insert will help us catch changes/regressions more easily?